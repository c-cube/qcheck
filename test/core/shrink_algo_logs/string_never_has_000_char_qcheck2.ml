open QCheck2

let string_never_has_000_char =
  Test.make ~name:"string never has a \\000 char"
    Gen.string
    (Log.shrinks_string @@
     fun s -> String.to_seq s |> Seq.fold_left (fun acc c -> acc && c <> '\000') true)
;;
try Test.check_exn string_never_has_000_char with Test.Test_fail _ -> ()
