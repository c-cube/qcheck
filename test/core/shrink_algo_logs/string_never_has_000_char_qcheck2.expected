holds "\137lk\019\159\153>\179\022Z\198\129\150\216\"\190\210\173n\234\245\196\005p\b\152A8@\021\156\r\244\248_0\166\235\168'\127\154$\025t\158\012\240\188L\238\212C\246\129\142\254\163?\202\167\207\187\250: \1836\192\207\223YH\149\024^u(X\191\213@\t\142\004\023\171\241\153\131\248G\182z,\178\162\222?(d\249<\243\229\001)\192\n\221T\186\255GX\165\239a\222\213\165\170\130\143M\239\b\245u\\m\2061\144\175\189[.]\021\246\244\173\146\230>\240\164\224\254\145^\128\164\204\185\218G\213pi\160\149@\\\249pCx\168\200\223lqV\160\236\1366/\228\167@\255\003H\127\012\199\142!\197\201\219@R\247\022\214d\212\187\141jj\231\229x;\021\226\165\236\031xi\244\025T${\134\"6\129\003+f\141\027\014\206P\190\152^\1607`TaO\184\193\174\175;\206\234\177"
holds "\248\177V\184\200\206\244\015\238I\127\205\129\007\1419\217"
fails "\011\241\027<r\r\130\187\151\200'\000\181,\019qi7\193\029\195\218L\210\186_"
holds ""
holds "\233\145\241\017Q\191T\145\205_\198\207j"
holds "h\147\007\b\150~\146P\247E_I\133\136\162=\210X\247\207"
fails "V\020\214t\235a\000hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds ""
holds "\br\166\156y\172\145\145\168\229\163"
holds "\224\\\238\015\129\001\127\016\027\225\1384\255\231\232O\005"
holds "\206\255C\226\182\189\\\236\151\243\148&K\187P\031\233Fp\161"
holds "\213\131\211\181n\214S\243_\255\135F\187\199wm\187\022\218\221\223"
holds "\151\175W4n4\204\227\158\151\197x-\199E\207\178\238\r\166\207*"
fails "a\020\214t\235a\000hl\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aa\214t\235a\000hl\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aaat\235a\000hl\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aaaa\235a\000hl\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaaahl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa1hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\rhl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002hl\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001hl\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000al\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaaaal\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa1al\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025al\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\ral\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007al\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004al\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002al\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001al\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaaaaa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa1aa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025aa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\raa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007aa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004aa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002aa\014z\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001aa\014z\200\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaaaaaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa1aaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025aaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\raaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007aaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004aaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002aaaz\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001aaaz\200\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaaaaaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa1aaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025aaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\raaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007aaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004aaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002aaaa\200\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001aaaa\200\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaaaaaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa1aaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025aaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\raaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007aaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004aaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002aaaaa\243,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001aaaaa\243,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaaaaaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa1aaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa\025aaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa\raaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa\007aaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa\004aaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa\002aaaaaa,\227\171Sg\172\211\244\183F"
holds "aaaaaa\001aaaaaa,\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaaaaaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa1aaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa\025aaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa\raaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa\007aaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa\004aaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa\002aaaaaaa\227\171Sg\172\211\244\183F"
holds "aaaaaa\001aaaaaaa\227\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaaaaaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa1aaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa\025aaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa\raaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa\007aaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa\004aaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa\002aaaaaaaa\171Sg\172\211\244\183F"
holds "aaaaaa\001aaaaaaaa\171Sg\172\211\244\183F"
fails "aaaaaa\000aaaaaaaaaSg\172\211\244\183F"
holds "aaaaaaaaaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa1aaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa\025aaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa\raaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa\007aaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa\004aaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa\002aaaaaaaaaSg\172\211\244\183F"
holds "aaaaaa\001aaaaaaaaaSg\172\211\244\183F"
fails "aaaaaa\000aaaaaaaaaag\172\211\244\183F"
holds "aaaaaaaaaaaaaaaaag\172\211\244\183F"
holds "aaaaaa1aaaaaaaaaag\172\211\244\183F"
holds "aaaaaa\025aaaaaaaaaag\172\211\244\183F"
holds "aaaaaa\raaaaaaaaaag\172\211\244\183F"
holds "aaaaaa\007aaaaaaaaaag\172\211\244\183F"
holds "aaaaaa\004aaaaaaaaaag\172\211\244\183F"
holds "aaaaaa\002aaaaaaaaaag\172\211\244\183F"
holds "aaaaaa\001aaaaaaaaaag\172\211\244\183F"
fails "aaaaaa\000aaaaaaaaaaa\172\211\244\183F"
holds "aaaaaaaaaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa1aaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa\025aaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa\raaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa\007aaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa\004aaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa\002aaaaaaaaaaa\172\211\244\183F"
holds "aaaaaa\001aaaaaaaaaaa\172\211\244\183F"
fails "aaaaaa\000aaaaaaaaaaaa\211\244\183F"
holds "aaaaaaaaaaaaaaaaaaa\211\244\183F"
holds "aaaaaa1aaaaaaaaaaaa\211\244\183F"
holds "aaaaaa\025aaaaaaaaaaaa\211\244\183F"
holds "aaaaaa\raaaaaaaaaaaa\211\244\183F"
holds "aaaaaa\007aaaaaaaaaaaa\211\244\183F"
holds "aaaaaa\004aaaaaaaaaaaa\211\244\183F"
holds "aaaaaa\002aaaaaaaaaaaa\211\244\183F"
holds "aaaaaa\001aaaaaaaaaaaa\211\244\183F"
fails "aaaaaa\000aaaaaaaaaaaaa\244\183F"
holds "aaaaaaaaaaaaaaaaaaaa\244\183F"
holds "aaaaaa1aaaaaaaaaaaaa\244\183F"
holds "aaaaaa\025aaaaaaaaaaaaa\244\183F"
holds "aaaaaa\raaaaaaaaaaaaa\244\183F"
holds "aaaaaa\007aaaaaaaaaaaaa\244\183F"
holds "aaaaaa\004aaaaaaaaaaaaa\244\183F"
holds "aaaaaa\002aaaaaaaaaaaaa\244\183F"
holds "aaaaaa\001aaaaaaaaaaaaa\244\183F"
fails "aaaaaa\000aaaaaaaaaaaaaa\183F"
holds "aaaaaaaaaaaaaaaaaaaaa\183F"
holds "aaaaaa1aaaaaaaaaaaaaa\183F"
holds "aaaaaa\025aaaaaaaaaaaaaa\183F"
holds "aaaaaa\raaaaaaaaaaaaaa\183F"
holds "aaaaaa\007aaaaaaaaaaaaaa\183F"
holds "aaaaaa\004aaaaaaaaaaaaaa\183F"
holds "aaaaaa\002aaaaaaaaaaaaaa\183F"
holds "aaaaaa\001aaaaaaaaaaaaaa\183F"
fails "aaaaaa\000aaaaaaaaaaaaaaaF"
holds "aaaaaaaaaaaaaaaaaaaaaaF"
holds "aaaaaa1aaaaaaaaaaaaaaaF"
holds "aaaaaa\025aaaaaaaaaaaaaaaF"
holds "aaaaaa\raaaaaaaaaaaaaaaF"
holds "aaaaaa\007aaaaaaaaaaaaaaaF"
holds "aaaaaa\004aaaaaaaaaaaaaaaF"
holds "aaaaaa\002aaaaaaaaaaaaaaaF"
holds "aaaaaa\001aaaaaaaaaaaaaaaF"
fails "aaaaaa\000aaaaaaaaaaaaaaaa"
holds "aaaaaaaaaaaaaaaaaaaaaaa"
holds "aaaaaa1aaaaaaaaaaaaaaaa"
holds "aaaaaa\025aaaaaaaaaaaaaaaa"
holds "aaaaaa\raaaaaaaaaaaaaaaa"
holds "aaaaaa\007aaaaaaaaaaaaaaaa"
holds "aaaaaa\004aaaaaaaaaaaaaaaa"
holds "aaaaaa\002aaaaaaaaaaaaaaaa"
holds "aaaaaa\001aaaaaaaaaaaaaaaa"
